### PR TITLE
Broker reconciler now makes a secret based on the config it points to

### PR DESCRIPTION
**Pull request by n3wscott**

### CODE REVIEW

File: pkg/reconciler/trigger/trigger.go [hunk 1]
```diff
@@ -20,6 +20,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	resources2 "knative.dev/eventing-rabbitmq/pkg/reconciler/broker/resources"
```
Comment (vaikas) [inline pkg/reconciler/trigger/trigger.go:23]: I'd prefer brokerresources?

File: pkg/reconciler/broker/resources/secret.go [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/eventing/pkg/apis/eventing"
+	"knative.dev/pkg/kmeta"
+)
+
+const (
+	BrokerURLSecretKey = "brokerURL"
+)
+
+// MakeIngress creates the in-memory representation of the Broker's ingress Deployment.
```
Comment (vaikas) [inline pkg/reconciler/broker/resources/secret.go:31]: nit, copy&pasta

File: pkg/reconciler/trigger/trigger.go [hunk 3]
```diff
@@ -230,7 +230,7 @@ func (r *Reconciler) reconcileDispatcherDeployment(ctx context.Context, t *v1bet
 		//ServiceAccountName string
 		RabbitMQSecretName: rabbitmqSecret.Name,
 		QueueName:          t.Name,
-		BrokerUrlSecretKey: broker.BrokerUrlSecretKey,
+		BrokerUrlSecretKey: resources2.BrokerURLSecretKey,
```
Comment (vaikas) [inline pkg/reconciler/trigger/trigger.go:233]: nit, BrokerURLSecretKey?

Comment (n3wscott) [reply pkg/reconciler/trigger/trigger.go:233]: ? I used `BrokerURLSecretKey`

**Pull request merged**